ord_schema/proto/build_database.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2020 Open Reaction Database Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Populates a PostgreSQL database containing the ORD.
+
+For simplicity, and to aid in debugging, we write tables to individual CSV
+files and load them into PostgreSQL with the COPY command.
+"""
+
+import collections
+import csv
+import glob
+import os
+
+from absl import app
+from absl import flags
+from absl import logging
+import psycopg2
+
+from ord_schema import message_helpers
+from ord_schema.proto import dataset_pb2
+from ord_schema.proto import reaction_pb2
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input', None, 'Input pattern (glob).')
+flags.DEFINE_string('output', 'tables', 'Output directory for CSV tables.')
+flags.DEFINE_string('database', None, 'Database name.')
```

Comment:
I think it's generally good to be explicit unless there's a compelling reason not to be (and it can make testing/debugging more difficult not to be). In this case the database write only happens if --database is set, so storing this in an env variable might not do quite what we want.